docsbuild/content/migrations/role.md

```diff
@@ -11,13 +11,18 @@ All migrations referring to the role resource.
 Add a role to keycloak, fails if the role already exists
 ### Parameter
 - realm: String, optional
-- name: String, not optional,
```

Comment:
Can we leave it as `not optional` cause if we change it here we should change it in the rest of documentation as well

Reply:
brought back "not optional"